vscode-web/main.tf

```
@@ -97,6 +97,30 @@ variable "settings" {
   default     = {}
 }
 
+variable "offline" {
+  type        = bool
+  description = "Just run VS Code Web in the background, don't fetch it from GitHub"
+  default     = false
+}
+
+variable "use_cached" {
+  type        = bool
+  description = "Uses cached copy of VS Code Web in the background, otherwise fetches it from internet"
```

The others have a period, so I added one here, but let me know if this is wrong.

```suggestion
  description = "Uses cached copy of VS Code Web in the background, otherwise fetches it from internet."
```